nano/core_test/active_transactions.cpp

```diff
@@ -731,7 +728,6 @@ TEST (active_transactions, republish_winner)
 					.build_shared ();
 		node1.process_active (fork);
 	}
-	node1.block_processor.flush ();
```

Comment:
Maybe here we need the use nano::test:exists()...

Reply:
Added.